scripts/configure-venue-access.ts

```diff
@@ -0,0 +1,141 @@
+import fs from "fs";
```

Comment:
This is missing the `#!` shebang from our latest script patterns. See others for an example.

Reply:
Is this the line you are referring to: `#!/usr/bin/env node -r esm -r ts-node/register`

Reply:
Yup, thats the one.

It's called a shebang:

- https://en.wikipedia.org/wiki/Shebang_(Unix)